server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlActionsMap.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ratelimitting.admissioncontrol;
+
+import org.opensearch.action.bulk.BulkAction;
+import org.opensearch.action.search.SearchTransportService;
+import org.opensearch.ratelimitting.admissioncontrol.enums.TransportActionType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Map to store the actions based on the type of the transport operation
+ */
+public class AdmissionControlActionsMap {
+    public static Map<String, String> transportActionsMap;
+
+    // Adding below as actions will add/update more actions based on discussions in next PR's.
+    static {
+        transportActionsMap = new HashMap<>();
+        transportActionsMap.put(BulkAction.NAME + "[s][p]", TransportActionType.INDEXING.getType());
```

Comment:
can we move [s][p] to constants file and use in all dependent files - instead of hardcoding

Reply:
Updated.